themes/bootstrap3/templates/search/facet-list-content.phtml

```diff
@@ -0,0 +1,22 @@
+<div class="full-facet-list facet-group<?php if(!$active): ?> hidden<?php endif; ?>" id="facet-list-<?=$this->escapeHtmlAttr($key) ?>">
+  <?php if ($active): ?>
+    <?php if ($this->page > 1): ?>
+      <a href="<?=$urlBase . '&amp;facetpage=' . ($this->page - 1) . '&amp;facetsort=' . urlencode($this->sort) ?>" class="facet js-facet-prev-page" data-page="<?=($this->page + 1) ?>" data-sort="<?=$this->sort ?>" data-limit="<?=count($this->data) ?>" data-lightbox-ignore><?=$this->translate('prev_ellipsis') ?></a>
+    <?php endif; ?>
+    <?php foreach ($this->data as $item): ?>
+      <?=$this->render('Recommend/SideFacets/single-facet.phtml', [
+        'exclude' => $this->exclude,
+        'facet' => $item,
+        'group' => $this->facet,
+        'url' => $this->results->getUrlQuery(),
+        'urlBase' => $this->searchAction
+      ]) ?>
+    <?php endforeach; ?>
+  <?php endif; ?>
+  <?php if ($this->anotherPage): ?>
+    <a href="<?=$urlBase . '&amp;facet=' . $facet . '&amp;page=' . ($this->page + 1) . '&amp;sort=' . urlencode($key) ?>" class="facet js-facet-next-page" data-page="<?=($this->page + 1) ?>" data-sort="<?=$this->escapeHtmlAttr($key) ?>">
```

Comment:
Note to self: make queries like this easier with MakeLink.

Reply:
I wasn't saying this was an issue with this PR, I was saying this kind of long query should be easier than it is and I want to add an option to MakeLink for it. Sorry for being vague!